content/docs/studio/user-guide/billing-and-payment.md

```diff
@@ -1,88 +1,83 @@
-collaboration functionality, you can sign up for the Team or Enterprise plans.
-These are paid plans, and in this section you will learn how to manage billing
-and payment for these plans.
+designed for teams that are starting out: you can invite one other team member
+and import as many repositories as you want.
 
-- [Difference between Free, Team and Enterprise plans](#difference-between-free-team-and-enterprise-plans)
+For more advanced collaboration, you can sign up for the Team or Enterprise
+plans. These are paid plans, and in this page you will learn how to manage
+billing and payment for them.
+
+- [Difference between Free, Team, and Enterprise plans](#difference-between-free-team-and-enterprise-plans)
 - [How to sign up for the Team or Enterprise plan](#how-to-sign-up-for-the-team-or-enterprise-plan)
 - [How total price is calculated](#how-total-price-is-calculated)
-- [How increasing or decreasing the number of seats affects total cost](#how-increasing-or-decreasing-the-number-of-seats-affects-total-cost)
+- [How increasing or decreasing the number of seats affects total price](#how-increasing-or-decreasing-the-number-of-seats-affects-total-price)
 - [Downgrading a plan](#downgrading-a-plan)
 - [Billing period, auto-renewal of team plan, and grace period for non-payment](#billing-period-auto-renewal-of-team-plan-and-grace-period-for-non-payment)
 - [Payment processor and accepted payment methods](#payment-processor-and-accepted-payment-methods)
 - [Accessing your payment history and invoices](#accessing-your-payment-history-and-invoices)
 - [Frequently asked questions](#frequently-asked-questions)
 
-## Difference between Free, Team and Enterprise plans
+## Difference between Free, Team, and Enterprise plans
 
 A detailed comparison of the different plans can be found in the Iterative
 Studio [pricing page](https://studio.iterative.ai/pricing).
 
 ## How to sign up for the Team or Enterprise plan
 
-When you sign up, you're on the Free plan by default. Plans are managed at the
-team level. You can
-[change the plan](/doc/studio/change-team-plan-or-size#change-team-plan) in a
-team's settings page. Each team is on its own plan and needs to be managed
-separately.
+When you sign up, you're on the **Free plan** by default. To switch to the
+**Team plan**, [create a team] first, then go to the [team's settings] and click
+on `Change plan` in the `Plan and billing` section. switch to the **Enterprise
+plan**, please [contact us].
+
+[contact us]: /doc/studio/troubleshooting#support
 
 ## How total price is calculated
 
 For Team plans, invoice amount is calculated based on the number of seats in the
 team. All invoice amounts are calculated for one billing period at a time. A
-billing period is one month long. The price per seat can be found in the
-Iterative Studio [pricing page](https://studio.iterative.ai/pricing).
-
-Below is an example calculation:
+billing period is one month long (30 days). The price per seat can be found in
```

Comment:
```suggestion
billing period is one month long. The price per seat can be found in
```
Reverting this bc/ I think it's wrong ☝🏼 but what happens if you sign up on a 31st? Or on Feb 28th? @tapadipti 🙏🏼